openml/flows/functions.py

```diff
@@ -94,11 +103,8 @@ def get_flow(flow_id: int, reinstantiate: bool = False) -> OpenMLFlow:
     ):
         flow = _get_flow_description(flow_id)
 
-    if reinstantiate:
-        if not (flow.external_version.startswith('sklearn==')
-                or ',sklearn==' in flow.external_version):
-            raise ValueError('Only sklearn flows can be reinstantiated')
-        flow.model = openml.flows.flow_to_sklearn(flow)
+    if reinstantiate and extension is not None:
```

Comment:
Does it make sense to drop the reinstantiate parameter? Providing the extension with `reinstantiate=False` does not use the extension at all, and providing no extension with `reinstantiate=True` does not reinstantiate the model (and without warning, no less).

Reply:
Good catch, I'll drop it.